meson.build

```diff
@@ -95,7 +95,7 @@ posix_console = posix_io and get_option('posix-console')
 sysroot_install = get_option('sysroot-install')
 newlib_obsolete_math = get_option('newlib-obsolete-math')
 
-host_cpu=host_machine.cpu()
+host_cpu=host_machine.cpu_family()
```

Comment:
Oh, thanks for this -- my cross compilation configuration sets cpu and cpu_family to be the same value, which
is not what meson wants. We should change 'host_cpu' to 'host_cpu_family' in all of the meson.build files to make
this change clear.

Reply:
> We should change 'host_cpu' to 'host_cpu_family' in all of the meson.build files to make

Done